src/controllers/user-controller.js

```diff
@@ -15,3 +17,19 @@ export const findAllUsers = (obj, args, { models }) => {
 export const findUserById = (_, { id }, { models }) => {
   return models.user.findByPk(id)
 }
+
+export const signIn = async (_, { data }, { models }) => {
+  const user =  models.user.findUserByUsername( data.username )
+  if (user){
+    const secret = process.env.JWT_SECRET;
```

Comment:
Use or not semicolons, but do it consistently in the code. At WeDevelop we follow StandardJS, which doesn't use semicolons.